BUILD.md

```diff
@@ -22,12 +22,12 @@
     * `cd /home/`
 
 10. Download projects
-    * `wget https://github.com/openssl/openssl/archive/OpenSSL_1_1_1.zip -O openssl.zip`
+    * `wget https://github.com/openssl/openssl/archive/openssl-3.0.11.zip -O openssl.zip`
     * `wget https://github.com/libimobiledevice/libplist/archive/2.2.0.zip -O libplist.zip`
     * `wget https://github.com/libimobiledevice/libusbmuxd/archive/2.0.2.zip -O libusbmuxd.zip`
     * `wget https://github.com/libimobiledevice/libimobiledevice/archive/1.3.0.zip -O libimobiledevice.zip`
     * `wget https://github.com/OutSystems/ios-webkit-debug-proxy/archive/outsystems.zip -O ios-webkit-debug-proxy.zip`
-    * `wget https://ftp.pcre.org/pub/pcre/pcre-8.43.zip -O pcre.zip`
+    * `wget https://altushost-swe.dl.sourceforge.net/project/pcre/pcre/8.43/pcre-8.43.zip -O pcre.zip`
```

Comment:
Why using a mirror of SourceForge and not the main site?

Reply:
I copied the wrong url. Fixed.